msteams-platform/TOC.yml

```
@@ -8,18 +8,18 @@
 - name: Get started
   expanded: true
   items:
-  - name: Overview
-    href: build-your-first-app/build-first-app-overview.md
-  - name: Build a "Hello, World!" app
-    href: build-your-first-app/build-and-run.md
-  - name: Build a basic personal tab
-    href: build-your-first-app/build-personal-tab.md
-  - name: Build your first channel and group tab
-    href: build-your-first-app/build-channel-tab.md
-  - name: Build your first bot
-    href: build-your-first-app/build-bot.md
-  - name: Build your first messaging extension
-    href: build-your-first-app/build-messaging-extension.md
+  - name: Prerequisites
+    href: get-started/prerequisites.md
+  - name: Create a personal tab app (React)
+    href: get-started/first-app-react.md
+  - name: Create a personal tab app (Blazor)
+    href: get-started/first-app-blazor.md
+  - name: Create a personal tab app (SPFx)
+    href: get-started/first-app-spfx.md
```

```suggestion
  - name: Create a personal app (React)
    href: get-started/first-app-react.md
  - name: Create a personal app (Blazor)
    href: get-started/first-app-blazor.md
  - name: Create a personal app (SPFx)
    href: get-started/first-app-spfx.md
```
See comments in other articles. Trying to better align what the end-user product is called (personal app) before getting into the details (tabs).

---

Approve - is in a bulk update coming at the end.